pkg/utils/file.go

```diff
@@ -25,7 +26,7 @@ func FileWalk(root string, walkFn func(r io.Reader, path string) error) error {
 		}
 
 		if info.Size() == 0 {
-			log.Printf("invalid size: %s\n", path)
+			log.Info("Invalid file size", log.String("path", path), log.Int64("size", info.Size()))
```

Comment:
I suggest to use `path` as constant (as in Trivy).
Also we can use `path` for `dir` (e.g. azure package)

Reply:
Do you mean adding a function?
https://github.com/aquasecurity/trivy/blob/f9b6d016854086f9c1edb96a1f70c248ae3c9324/pkg/log/handler.go#L281-L284

Reply:
yes, make a function.
Sorry for the confusion

Reply:
But perhaps these are extra changes 😄

Reply:
They are not user-facing messages. IMHO, message consistency is no big deal, but there's no downside of defining functions. Added in https://github.com/aquasecurity/trivy-db/pull/486/commits/518f5629d9bef210dac3c3ac1d7ed08acaf7783b